modules/social_features/social_group/config/install/social_group.settings.yml

```diff
@@ -1 +1,2 @@
+create: true
```

Comment:
@OH17L Please rename this property to something like `allow_group_create` to be consistent with existing setting, because `create` is too general.

Reply:
Done